aws/sdk/integration-tests/s3/tests/endpoints.rs

```diff
@@ -62,24 +62,19 @@ async fn dual_stack() {
     );
 }
 
+#[cfg(feature = "sigv4a")]
 #[tokio::test]
 async fn multi_region_access_points() {
-    let (_captured_request, client) = test_client(|b| b);
-    let response = client
+    let (captured_request, client) = test_client(|b| b);
+    let _ = client
         .get_object()
         .bucket("arn:aws:s3::123456789012:accesspoint/mfzwi23gnjvgw.mrap")
         .key("blah")
         .send()
         .await;
-    let error = response.expect_err("should fail—sigv4a is not supported");
-    assert!(
-        dbg!(format!(
-            "{}",
-            aws_smithy_types::error::display::DisplayErrorContext(&error)
-        ))
-        .contains("selected auth scheme / endpoint config mismatch"),
-        "message should contain the correct error, found: {:?}",
-        error
+    assert_eq!(
```

Comment:
can we also ensure that the header is set and looks right? (obviously we can't check the actual signature but I assume it has `sigv4a` in there or something?)